### ISSUE TITLE
Test for WCAG technique G171: Playing sounds only on user request

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#208]
http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/G171
The intent of this technique is to allow a user to control the use of sounds in Web content. Someone that uses a screen reader may find it very distracting and difficult to listen to their screen reader if there are also sounds coming from Web content. Providing a way to play sounds only upon request will give a user the control needed to listen to any sounds or other audio without interfering with the output from a screen reader.
## Test procedure
1. Load a Web page that is known to contain sounds that play for 3 seconds or longer.
2. Check that no sounds play automatically.
3. Check that there is a way for a user to start sounds manually.
## Applicable

Video and audio content
## Pass

Audio or video not automatically played
html5: foreach mediaelement check that is has property muted or paused
objects: check with their javascript API if the video is muted or paused.
## Fail

Audio or video is automatically played, note that then the other techniques becoming applicable.
